libc/include/llvm-libc-macros/langinfo-macros.h

```diff
@@ -0,0 +1,81 @@
+//===-- Definition of macros from langinfo.h ------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_MACROS_LANGINFO_MACROS_H
+#define LLVM_LIBC_MACROS_LANGINFO_MACROS_H
+
+// TODO: Define these for the C locale.
```

Comment:
If these are going to be hardcoded to the default locale, is it worthwhile to just set the obvious ones now? Like `AM_STR` is going to be `"AM"`, etc.

Reply:
I agree, the C locale is most likely going to be the only one we're going to support in LLVM libc so we might as well make it usable right away.